_posts/2018-07-25-the_incredibles_3.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: post
+title: "The Incredibles 3"
+date: "2018-07-25 21:12:49 -0500"
+---
+
+I've been thinking a lot about _The Incredibles 2_ since I saw it. Now that we
+have two entries in the series, we can make some statements about what subjects
+the franchise as a whole addresses. At least if we're comfortable extrapolating
+a line from two points. I'm comfortable with doing that and so I'd like to talk
+about some of those topics and how I'd like to see them developed in a
+theoretical third movie: _The Incredibles 3_.
+
+
+## Content Warning
+
+Expect spoilers for _The Incredibles_ and _The Incredibles 2_ below. Not a ton,
+but still.
+
+Also, I'm doing a somewhat feminist reading of the franchise and below I'm going
+to lightly touch on racism and situations where men emotionally manipulate
```

Comment:
split infinitive

Reply:
Unsplit infinitives are for people who think the Romans were infallible in all things and that English should be Latin. 😜